projects/storefrontlib/src/lib/cms/services/cms-routes.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@angular/core';
+import { Route, Router } from '@angular/router';
+import { CmsRoute, CmsService, PageContext, PageType } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { PageLayoutComponent } from '../page-layout/page-layout.component';
+import { CmsMappingService } from './cms-mapping.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CmsRoutesService {
+  constructor(
+    private router: Router,
+    private cmsService: CmsService,
+    private cmsMapping: CmsMappingService
+  ) {}
+
+  contentRouteExist(url: string): boolean {
```

Comment:
It's rather dynamically added, cms driven route. Not content route

Reply:
👍